$OpenBSD: patch-src_Preferences_cpp,v 1.3 2012/02/22 07:54:26 dcoppa Exp $

Disable checking for new versions at startup.

Change nodes.dat default download URL to something actually working
(upstream git commit 8dbe597539bcac7272ccf4d46790fde814aeded1)

--- src/Preferences.cpp.orig	Sun Oct  9 02:19:08 2011
+++ src/Preferences.cpp	Tue Feb 21 15:58:53 2012
@@ -1210,7 +1210,7 @@ void CPreferences::BuildItemList( const wxString& appd
 	/**
 	 * Version check
 	 **/
-	 NewCfgItem(IDC_NEWVERSION,	(new Cfg_Bool( wxT("/eMule/NewVersionCheck"), s_NewVersionCheck, true )));
+	 NewCfgItem(IDC_NEWVERSION,	(new Cfg_Bool( wxT("/eMule/NewVersionCheck"), s_NewVersionCheck, false )));
 
 	 /**
 	  * Obfuscation
@@ -1244,7 +1244,7 @@ void CPreferences::BuildItemList( const wxString& appd
 
 	s_MiscList.push_back( new Cfg_Bool( wxT("/eMule/DropSlowSources"), 		s_DropSlowSources, false ) );
 
-	s_MiscList.push_back( new Cfg_Str(  wxT("/eMule/KadNodesUrl"),			s_KadURL, wxT("http://download.tuxfamily.org/technosalad/utils/nodes.dat") ) );
+	s_MiscList.push_back( new Cfg_Str(  wxT("/eMule/KadNodesUrl"),			s_KadURL, wxT("http://upd.emule-security.org/nodes.dat") ) );
 	s_MiscList.push_back( new Cfg_Str(  wxT("/eMule/Ed2kServersUrl"),		s_Ed2kURL, wxT("http://gruk.org/server.met.gz") ) );
 	s_MiscList.push_back( MkCfg_Int( wxT("/eMule/ShowRatesOnTitle"),		s_showRatesOnTitle, 0 ));
 
